.editorconfig

```diff
@@ -0,0 +1,13 @@
+# editorconfig.org
+root = true

```

Comment:
why this change? I guess this file should be removed

Reply:
it defines setups for editors (analogue of .vscode/settings.json, but without reference to a specific one)

Reply:
ok why not:)

you have some checks failing and I can merge